Library/Formula/gupnp.rb

```diff
@@ -27,6 +27,11 @@ class Gupnp < Formula
   depends_on "libsoup"
   depends_on "gssdp"
 
+  patch :p0 do
+    url "https://trac.macports.org/export/136642/users/devans/GNOME-3/stable/dports/net/gupnp/files/patch-osx-uuid.diff"
```

Comment:
Sigh. Thanks for kicking it upstream, you may need to kick it to them directly as well, I'm not sure how actively they check the Github mirrors. Happy to merge it with the reference to the PR though.